.rubocop.yml

```diff
@@ -61,6 +70,9 @@ Layout/IndentFirstHashElement:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/NumericPredicate:
+  Enabled: false
```

Comment:
There were ~5 violations. The challenge I had was that that that when doing `something == 0` I wasn't sure whether `something` would always be a number, because it's not the same behavior if `something` can be `nil` or, for example, a string. 

...so ignored it. 

I think `.zero?` is more readable when it's guaranteed the object will respond to it.

Reply:
Ah, yeah, I hear you if you aren’t confident about types. My concerns come more from `positive?` and `negative?`, which I always find ambiguous with respect to whether zero is included, and similar functions in different languages have differing opinions there (especially in languages that support `-0` as a value). But anyway, doesn’t really matter here for now.